Sources/Testing/Events/Event.swift

```diff
@@ -15,8 +15,11 @@ public struct Event: Sendable {
   public enum Kind: Sendable {
     /// A test run started.
     ///
+    /// - Parameters:
+    ///   - plan: The test plan of the run that started.
+    ///
     /// This event is the first event posted after ``Runner/run()`` is called.
-    case runStarted
+    indirect case runStarted(_ plan: Runner.Plan)
```

Comment:
I suspect this is added because it's used elsewhere, but isn't this quite a large payload potentially? I wonder about the impact of that

But also (ironically): if we do add this here, for parity should we also be adding it to the corresponding _run ended_ event?

Reply:
It's pretty chunky, yes, but remember that the value is CoW, so not as large as you might think. I'm open to alternative designs.

I left it out of the `runEnded` case precisely because it was relatively large.